clinica/pipelines/pet_volume/pet_volume_pipeline.py

```diff
@@ -1,8 +1,9 @@
 # coding: utf8
```

Comment:
You need to change call of `check_relative_volume_location_in_world_coordinate_system` after file grabbing that uses `pet_type` key instead of `tracer` (I cannot comment on it since you did not change it)

Reply:
Thanks ! :) Fixed in 70c6718